<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readEDF</class>
 <widget class="QDialog" name="readEDF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>readEDF</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>580</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of samples:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="chList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Date:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-----&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-----&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sample frequency:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-----&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineFrom">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;File duration:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="filedurationLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-----&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="freqLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-----&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Recording:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="nOfSamplesLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-----&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Channels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="recordingLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-----&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>20</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Files added</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channels chosen for File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>390</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Read file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>File Info</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Queued files info</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="filesAdded">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="channelsForFile">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="filesLoaded">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>791</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addAllFilesButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add all files</string>
   </property>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>label_7</zorder>
  <zorder>chList</zorder>
  <zorder>label_4</zorder>
  <zorder>dateLabel</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>timeLabel</zorder>
  <zorder>label_10</zorder>
  <zorder>nameLabel</zorder>
  <zorder>lineFrom</zorder>
  <zorder>label_6</zorder>
  <zorder>filedurationLabel</zorder>
  <zorder>freqLabel</zorder>
  <zorder>label_3</zorder>
  <zorder>label_12</zorder>
  <zorder>nOfSamplesLabel</zorder>
  <zorder>label</zorder>
  <zorder>recordingLabel</zorder>
  <zorder>lineTo</zorder>
  <zorder>label_11</zorder>
  <zorder>line_3</zorder>
  <zorder>label_9</zorder>
  <zorder>label_13</zorder>
  <zorder>addButton</zorder>
  <zorder>removeButton</zorder>
  <zorder>readFileButton</zorder>
  <zorder>label_14</zorder>
  <zorder>line_4</zorder>
  <zorder>label_15</zorder>
  <zorder>line_5</zorder>
  <zorder>filesAdded</zorder>
  <zorder>channelsForFile</zorder>
  <zorder>filesLoaded</zorder>
  <zorder>label_16</zorder>
  <zorder>addAllFilesButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>readEDF</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>readEDF</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
